udata_front/theme/gouvfr/assets/js/pages/ReusePublishingForm/Step2AddDatasets.vue

```diff
@@ -0,0 +1,194 @@
+          <div class="fr-col-auto fr-my-auto fr-ml-auto justify-center flex fr-mr-3v">
+            <img :src="draggableIcon" />
+          </div>
+          <div class="fr-col relative">
+            <DatasetCard
+              :dataset="dataset"
+              :dataset-url="dataset.slug"
+            />
+          </div>
+          <button
+            type="button"
+            class="fr-btn fr-btn--tertiary-no-outline fr-btn--secondary--error fr-icon-close-line border-0 absolute right-1px fr-mr-3w fr-mt-5w"
+            @click="removeDataset(index)"
+          ></button>
+        </div>
+      </div>
+      <MultiSelect
+        :minimumCharacterBeforeSuggest="2"
+        :placeholder="t('Look for a dataset')"
+        :searchPlaceholder="t('Search a dataset...')"
+        suggestUrl="/datasets/suggest/"
+        @change="handleDatasetChange"
+        :hasError="fieldHasError('datasets')"
+        :errorText="getErrorText('datasets')"
+      />
+      <p class="fr-hr-or w-100 text-transform-lowercase fr-text--regular">
+        <span class="fr-hr-or-text">{{ t('or') }}</span>
+      </p>
+      <InputGroup
+        :label="t('Link to the dataset')"
+        :placeholder="'https://...'"
+        class="w-100"
+        v-model="linkedDataset"
+        @change="handleLinkedDatasetChange"
+        :hasError="fieldHasError('linkedDataset')"
+        :hasWarning="fieldHasWarning('linkedDataset')"
+        :errorText="t('No dataset has been found on the provided url')"
+      />
+    </fieldset>
+    <Alert type="error" v-if="errors.length" class="fr-mb-2w">
+      <template #title>{{ t("An error occured | Some errors occured", errors.length) }}</template>
+      <ul v-if="errors.length > 1">
+        <li v-for="error in errors">{{ error }}</li>
+      </ul>
+      <p v-else> {{ errors[0] }}</p>
+    </Alert>
+    <div class="fr-grid-row fr-grid-row--right">
+      <button class="fr-btn" @click="submit">
+        {{ t("Next") }}
+      </button>
+    </div>
+  </Container>
+</template>
+<script setup lang="ts">
+import { computed, ref, toValue } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { type Dataset, DatasetCard, Well } from '@etalab/data.gouv.fr-components';
+import { useSortable } from '@vueuse/integrations/useSortable';
+import Container from '../../components/Ui/Container/Container.vue';
+import InputGroup from '../../components/Form/InputGroup/InputGroup.vue';
+import MultiSelect from '../../components/MultiSelect/MultiSelect.vue';
+import Stepper from '../../components/Form/Stepper/Stepper.vue';
+import Alert from '../../components/Alert/Alert.vue';
+import useFunctionalState from '../../composables/form/useFunctionalState';
+import { requiredWithCustomMessage } from '../../i18n';
+import { api } from '../../plugins/api';
+import draggableIcon from "../../../../templates/svg/illustrations/draggable.svg";
+
+const props = defineProps<{
+  errors: Array<string>,
+  steps: Array<string>,
+  loading?: Boolean,
```

Comment:
```suggestion
  loading?: boolean,
```
Always use primitive (lowercase) type and not their object (capitalize) type